exporter/prometheusremotewriteexporter/testutil_test.go

```diff
@@ -496,3 +548,14 @@ func getTimeSeries(labels []prompb.Label, samples ...prompb.Sample) *prompb.Time
 		Samples: samples,
 	}
 }
+
+func getQuantiles(bounds []float64, values []float64) []*otlp.DoubleSummaryDataPoint_ValueAtQuantile {
+	var quantiles []*otlp.DoubleSummaryDataPoint_ValueAtQuantile
```

Comment:
Maybe a nit but based on a recent comment from Bogdan on one of my own PR, the quantiles array can be preallocated and then each index can simply be filled

ie. 

```
quantiles := make([]*otlp.DoubleSummaryDataPoint_ValueAtQuantile, len(bounds))
for i := 0; i < len(bounds); i++ {
  quantiles[i] = ...
}
```